### PR TITLE
chore: version bump to 2.1.0 and changelog

### PR DESCRIPTION
**Pull request by polomani**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,18 +2,27 @@
 
 This article documents the ongoing improvements we're making to the **Cognite Data Source for Grafana**.
 
+## 2.1.0 - November 18, 2020
+
+* The data source requires Grafana version `7.0.6` or above.
+* Time series are now saved to dashboard by its `externalId` where possible.
```
Comment (arnekv) [inline CHANGELOG.md:8]:
```suggestion
* Time series are now saved to dashboards by their  `externalId` where possible.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,18 +2,27 @@
...
+This allows to switch between CDF projects that contain time series with matching `externalIds`.
```
Comment (arnekv) [inline CHANGELOG.md:9]:
```suggestion
This allows you to switch between CDF projects that contain time series with matching `externalIds`.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,18 +2,27 @@
...
+* Support [synthetic time series alignment](https://docs.cognite.com/dev/concepts/resource_types/synthetic_timeseries.html#alignment)
```
Comment (arnekv) [inline CHANGELOG.md:10]:
```suggestion
* Support for [synthetic time series alignment](https://docs.cognite.com/dev/concepts/resource_types/synthetic_timeseries.html#alignment).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,18 +2,27 @@
...
+* Rewrote data source GUI to React under the hood.
```
Comment (arnekv) [inline CHANGELOG.md:11]:
```suggestion
* Rewrote data source GUI to React.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,18 +2,27 @@
...
+* Updated CDF logo to match light/dark theme, [signed source code](https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin/)
```
Comment (arnekv) [inline CHANGELOG.md:12]: I'm not sure about the link here? How is signed source code relevant to an updated CDF logo?

**Review by arnekv — APPROVED** [review]
I've suggested a few changes + added a question.

File: CHANGELOG.md (hunk 5 above)
Comment (polomani) [reply CHANGELOG.md]: removed it, thanks

**Review by tengelskar — APPROVED** [review]

**Pull request merged**